pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -71,6 +71,8 @@ const LibvirtLocalConnectionPort = 22222
 const gpuEnvPrefix = "GPU_PASSTHROUGH_DEVICES"
 const vgpuEnvPrefix = "VGPU_PASSTHROUGH_DEVICES"
 
+var cloudInitDataStore *cloudinit.CloudInitData
```

Comment:
this probably belongs on the LibevirtDomainManager struct and I think you'll want to protect it with the domainModifyLock. 

you might not need this though.

Reply:
thanks, I've moved it to the LibvirtDomainManager. 
Why do you think I don't need that?
I mean, I think could avoid storing this, but I'm not sure about 

> cloudInitData, err = hooksManager.PreCloudInitIso(vmi, cloudInitData)

This should probably happen before the domain definition...?